src/taxpasta/infrastructure/application/__init__.py

```diff
@@ -18,3 +18,7 @@
 from .kraken2_profile_standardisation_service import (
     Kraken2ProfileStandardisationService,
 )
+from .metaphlan_profile_reader import MetaphlanProfileReader
+from .metaphlan_profile_standardisation_service import (
+    MetaphlanProfileStandardisationService,
+)
```

Comment:
Should also make available the profile itself.

```suggestion
from .metaphlan_profile import MetaphlanProfile
from .metaphlan_profile_reader import MetaphlanProfileReader
from .metaphlan_profile_standardisation_service import (
    MetaphlanProfileStandardisationService,
)
```